WcaOnRails/app/models/user.rb

```diff
@@ -78,7 +78,9 @@ def wca_id_is_unique_or_for_dummy_account
       user = User.find_by_wca_id(wca_id)
       # If there is a non dummy user with this WCA ID, fail validation.
       if user && !user.dummy_account?
-        errors.add(:wca_id, I18n.t('users.errors.unique'))
+        errors.add(:wca_id, I18n.t('users.errors.unique',
+                   used_name: user.name,
```

Comment:
Travis is picky about alignment, and this is making him unhappy =( See https://travis-ci.org/thewca/worldcubeassociation.org/builds/298264989#L861